src/web/scraping/unique.md

```diff
@@ -9,27 +9,20 @@ look for all entries of internal and external links with
 MediaWiki link syntax is described [here][MediaWiki link syntax].
 
 ```rust,no_run
-# #[macro_use]
-# extern crate error_chain;
 #[macro_use]
 extern crate lazy_static;
 extern crate reqwest;
 extern crate regex;
 
-use std::io::Read;
 use std::collections::HashSet;
 use std::borrow::Cow;
+use std::error::Error;
 use regex::Regex;
 
-# error_chain! {
-#     foreign_links {
-#         Io(std::io::Error);
-#         Reqwest(reqwest::Error);
-#         Regex(regex::Error);
-#     }
-# }
-#
-fn extract_links(content: &str) -> Result<HashSet<Cow<str>>> {
+
+#[tokio::main]

```

Comment:
Doesn't this only work right above the main function?

Reply:
Nop I tried it , because i expected that and not :S